ArduPlane/mode_manual.cpp

```diff
@@ -7,6 +7,25 @@ void ModeManual::update()
     SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, plane.pitch_in_expo(false));
     plane.steering_control.steering = plane.steering_control.rudder = plane.rudder_in_expo(false);
 
+    SRV_Channels::set_output_scaled(SRV_Channel::k_rudder, plane.rudder_in_expo(false));
```

Comment:
We use `plane.rudder_in_expo(false)` on line 8, you could stash it rather than fetching it twice. eg
```
const float pilot_rudder = plane.rudder_in_expo(false);
```
Then use that value on line 8 and 10.